### PR TITLE
feat(packager): reduce options to simplify docker usage

### PR DESCRIPTION
**Pull request by akantcheff**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -194,9 +187,8 @@ func buildTomcatBundle() {
 		}
 	}
 	fmt.Println("\nSuccessfully re-packaged self-contained application:", filepath.Join("output", bundleName+"-application.zip"))
-	fmt.Println("To use it, simply unzip it like your usual Bonita Tomcat bundle, and run ./start-bonita[.sh|.bat]")
+	fmt.Println("\nTo use it, simply unzip it like your usual Bonita Tomcat bundle, and run ./start-bonita[.sh|.bat]")
```
Comment (educhastenier) [inline main.go:190]: There is already a newline in Print**ln**

**Review by educhastenier — COMMENTED** [review]
A general remark:
With this simplification, SP users MUST know the base Subscription image name and version and pass it as parameter, otherwise they will have a Community image created.
Which is more difficult for them

File: main.go (hunk 1 above)
Comment (akantcheff) [reply main.go:190]: To make it coherent with the Docker usage: the final message with the final output is distinctly visible in the stdout.

**Comment by akantcheff** [comment]
> A general remark: With this simplification, SP users MUST know the base Subscription image name and version and pass it as parameter, otherwise they will have a Community image created. Which is more difficult for them

We discussed about it with Pascal and Romain. In fact the `-bonita-base-name` option should be mandatory with no default value (I let it for a convenient usage). A user should know at least which version its application was built in to be compliant.

However I can add an explanation in README for SP users on how to use Quay (deprecated) and incoming Artifactory.

File: main.go (hunk 1 above)
Comment (educhastenier) [reply main.go:190]: ok, fine

**Review by passga — APPROVED** [review]
LGTM!!!

**Pull request merged**